src/js/helpers/disasterHelper.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * disasterHelper.js
+ * Created by Jonathan Hill 06/11/20
+ */
+
+import { apiRequest } from 'helpers/apiRequest';
+
+const defcAPI = () => apiRequest({
```

Comment:
Maybe call this something like `fetchDefc` to be more consistent with other api request helpers?